tools/checkpoint_loader_megatron.py

```diff
@@ -58,6 +58,10 @@ def _load_checkpoint(queue, args):
         sys.argv += ["--bf16"]
 
     margs = megatron.arguments.parse_args()
+
+    if args.load_iters is not None:
+        margs.load_iters = args.load_iters
+
     margs = load_args_from_checkpoint(margs)
```

Comment:
To keep it consisten with the way we deal with other special options, I would suggest the `--load_iters` argument to be appended to `sys.argv` instead of `margs` directly.

```suggestion
        sys.argv += ["--bf16"]
    if args.load_iteration is not None:
        sys.argv += ["--load_iters", args.load_iters]

    margs = megatron.arguments.parse_args()
    margs = load_args_from_checkpoint(margs)
```